src/main/cpp/VideoState.cpp

```diff
@@ -1670,6 +1679,10 @@ double VideoState::get_master_clock() const {
 	return val;
 }
 
+double VideoState::get_fps() const {
+	return video_st ? this->fps : 0;
```

Comment:
Maybe return -1 to make it consistent with what happens in java if we have an exception?